project/plugin.sbt

```diff
@@ -1 +1 @@
-addSbtPlugin("org.foundweekends.giter8" % "sbt-giter8" % "0.9.0")
+addSbtPlugin("org.foundweekends.giter8" % "sbt-giter8" % "0.7.1")
```

Comment:
Why did we need to downgrade plugin?

Reply:
Sorry was trying some things to fix the build and forgot to put 0.9.0 back in. Ive reverted that change